crawler/cv/spiders/techcrunch.py

```diff
@@ -29,6 +29,10 @@ class TechcrunchSpider(scrapy.Spider):
     def parse(self, response):
         # TODO parse homepage for update
         # TODO parse list page get the article url
+        if re.compile('.*\/page\/\d+\/.*').match(response.url) is not None:
```

Comment:
Regular as a last option. Shouldn't be considered prior to `xpath` or `css` selector?